compiler/src/parsing/ast_iterator.re

```diff
@@ -118,15 +121,28 @@ module E = {
     | PExpApp(e, el) =>
       sub.expr(sub, e);
       List.iter(sub.expr(sub), el);
-    | PExpConstruct(c, el) =>
+    | PExpConstruct(c, e) =>
       iter_ident(sub, c);
-      List.iter(sub.expr(sub), el);
+      switch (e) {
+      | PExpConstrTuple(el) => List.iter(sub.expr(sub), el)
```

Comment:
It is done in 5 places by my count; I suppose a new function could be made for this but it wouldn't be adding much in readability imo since it's a pretty short snippet.

Reply:
The snippet is short but `iter_expressions` (or whatever it would be called) might just be nicer to read 🤔